### PR TITLE
Docs improvement

### PR DESCRIPTION
**Pull request by TheAwiteb**
Some improvements

### CODE REVIEW

**Comment by TheAwiteb** [comment]
image

I like it

File: README.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
+<div align="center">
+
 ![Group 1](https://user-images.githubusercontent.com/42001064/198829818-c4035432-8721-45e1-ba2d-4d2eb6d0c584.svg)
 
-Blazingly fast file search library built in Rust 🔥 [Work in progress]
+Blazingly fast file search crate built in Rust 🔥
 
 [![Version info](https://img.shields.io/crates/v/rust_search.svg)](https://crates.io/crates/rust_search)
+[![Documentation](https://docs.rs/rust_search/badge.svg)](https://docs.rs/rust_search)
+[![License](https://img.shields.io/crates/l/rust_search.svg)](https://github.com/rohitjmathew/rust_search/blob/master/LICENSE-MIT)
```
Comment (ParthJadhav) [inline README.md]:
```suggestion
[![License](https://img.shields.io/crates/l/rust_search.svg)](https://github.com/parthjadhav/rust_search/blob/master/LICENSE-MIT)
```

Maybe you mean't this...

File: src/builder.rs [hunk 2]
```diff
@@ -80,90 +80,94 @@ impl SearchBuilder {
...
+    ///     .build()
+    ///     .collect();
     /// ```
     pub fn ignore_case(mut self) -> Self {
         self.ignore_case = true;
         self
     }
 
-    /// Searches for hidden files
+    /// Searches for hidden files, meaning files that start with a dot.
```
Comment (ParthJadhav) [inline src/builder.rs]:
```suggestion
    /// Searches for hidden files.
```

I don't think that only files which have a dot are hidden...

**Review by ParthJadhav — APPROVED** [review]
Just a few suggestions

**Comment by ParthJadhav** [comment]
Yes, Centred Div looks good 😄

**Pull request merged**